source/include/defender.h

```diff
@@ -28,6 +28,12 @@
 #ifndef DEFENDER_H_
 #define DEFENDER_H_
 
+/* *INDENT-OFF* */
+#ifdef __cplusplus
+    extern "C" {
+#endif
+/* *INDENT-ON* */
+
 /* Standard includes. */
 #include <stdint.h>
```

Comment:
Oscar and I did after standard library includes, not that it matters that much since the standard library should already have this
```suggestion
/* Standard includes. */
#include <stdint.h>

/* *INDENT-OFF* */
#ifdef __cplusplus
    extern "C" {
#endif
/* *INDENT-ON* */
```